src/actions/sources.js

```diff
@@ -220,6 +220,11 @@ export function selectLocation(location: LocationObject) {
 
     dispatch(addTab(source.toJS(), 0));
 
+    if (source.get("loadedState") == "unloaded") {
+      if (!location.line) location.line = 1;
+      if (!location.column) location.column = 1;
```

Comment:
maybe we just do `location = {...location, line: 1, column: 1}`

Reply:
Checked, it bears different results as we are unconditionally rewriting line and column. For example, when someone clicks a link to follow a source that is not yet loaded the above would overwrite the line and column the person wanted to go to.